isso/js/app/templates/comment-loader.js

```diff
@@ -0,0 +1,9 @@
+var html = function (globals) {
+  // De-structure globals dict
+  ({comment, pluralize} = {...globals});
+  return "" +
+"<div class='isso-comment-loader' id='isso-loader-${comment.name}'>"
++ "<a class='load_hidden' href='#'>" + pluralize('comment-hidden', comment.hidden_replies) + "</a>"
++ "</div>"
```

Comment:
Wouldn't had be cleaner to use template string for this?

```js
return `
  <div class='isso-comment-loader' id='isso-loader-${comment.name}'>
    <a class='load_hidden' href='#'>${pluralize('comment-hidden', comment.hidden_replies)}</a>
  </div>
`
```

Reply:
While writing this, I see that you forgot to interpolate `${comment.name}`! Be careful.

Reply:
While writing this, I see that you forgot to interpolate `${comment.name}`! Be careful.

Reply:
> Wouldn't had be cleaner to use template string for this?

I did consider using template literals, see https://github.com/ix5/isso/tree/template-literals, but I saw browser compat wasn't too great, see [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#browser_compatibility)

But if we're using destructuring assignments, that argument kind of goes out of the window anyway.

> While writing this, I see that you forgot to interpolate ${comment.name}! Be careful.

Yup, way ahead of you ;) https://github.com/posativ/isso/commit/727dbc333dcef71aa1b9d4571021769223290291

Reply:
For your question, I would have done the same but extracting some repetitions into variables before the template string. This way the template is a bit cleaner. For example you are using 2 times ``${comment.author ? comment.author : i18n('comment-anonymous')}`. I would have done: 

```js
const author = comment.author ? comment.author : i18n('comment-anonymous');

return `
  <span class='author'>${author}</span>
`;
```

Reply:
Doh! That's so obvious in hindsight... Thank you.